test/Gopkg.toml

```diff
@@ -1,7 +1,7 @@
 # Force dep to vendor the code generators, which aren't imported just used at dev time.
 required = [
-  "github.com/knative/test-infra/scripts",
-  "github.com/knative/test-infra/tools/dep-collector",
+  "knative.dev/test-infra/scripts",
+  "knative.dev/test-infra/tools/dep-collector",
```

Comment:
``
[prune]
  go-tests = true
  non-go = true
  unused-packages = true
[[prune.project]]
  name = "github.com/knative/test-infra"
  non-go = false
```

The prune setting isnt applied causing the script `test/vendor/knative.dev/test-infra/scripts/e2e-tests.sh` to be deleted and test fails
```
./test/lib.bash: line 42: go_test_e2e: command not found
  ERROR 14:37:09.808 Tests have failures!
```

need to change
```
[[prune.project]]
  name = "github.com/knative/test-infra"
```
part to `knative.dev/test-infra`

Reply:
ugh, thanks!